### ISSUE TITLE
Crashed and I lost all of my progress

### ISSUE THREAD

**Issue by Ispenttoomuchtimemakingausername** [photopea/photopea#3562]
I was trying to group together some layers to duplicate them and it just crashed and I lost all of my progress. Not happy.

**Comment by photopea** [comment]
I am sorry to hear that. But it is a web browser who "kills" a website, probably because it ran out of memory.

Can you update your web browser? Otherwise, if you know you have very little RAM memory, save your work as a PSD at least once an hour.

**Comment by Ispenttoomuchtimemakingausername** [comment]
I have 32GB Ram. I think Chrome is to date. Do you recommend a different browser?

**Comment by Ispenttoomuchtimemakingausername** [comment]
Found out what happened since It did it again. It keeps telling me to disable my ad blocker but I have already done that.

**Comment by photopea** [comment]
We have an article about ads here: https://www.photopea.com/tuts/ads-in-photopea-do-not-work-what-to-do/

Could you describe steps, how to make Photopea crash?

**Comment by Ispenttoomuchtimemakingausername** [comment]
1: Install adblock
2: don't disable on all sites before loading photopea

**Comment by photopea** [comment]
Do you think your ad block makes Photopea crash? Isn't that a bug in the ad block instead of Photopea?

**Comment by Ispenttoomuchtimemakingausername** [comment]
It's because of the pop up telling me to disable the adblock

**Comment by photopea** [comment]
You should either disable your ad block, or pay for Premium. Otherwise, you will always see this message.

**Comment by Ispenttoomuchtimemakingausername** [comment]
Yes and I have disabled to ad block, but it only works if you reload the site. I am letting it show ads now

**Comment by photopea** [comment]
Photopea tries to load ads only once at the beginning. So ads should be allowed right when you start Photopea.

**Comment by Ispenttoomuchtimemakingausername** [comment]
Yep that's fine

**Comment by photopea** [comment]
Thanks!
